css/shopping-cart.css

```diff
@@ -115,4 +116,602 @@
   .shopping-cart__bottom-right {
     margin-top: 0; 
   }
+}
+
+
+/* Movil style */
+.purchase-summary-top{
+  max-width: 100%;
+  width: var(--width);
+  padding: var(--zero);
+  margin: 0 auto;
+}
+
+.purchase-summary__background{
+  background-color: var(--color-white);
+  box-shadow: var(--box-shadowL);
+}
+
+.purchase-summary__send{
+  display: flex;
+  gap: var(--eight-pixel);
+  width: var(--width);
+  height: auto;
+  align-items: flex-start;
+  margin-left: 0;
+  padding: var(--twenty-pixel);
+}
+
+.purchase-summary__send-text{
+  font-size: var(--font-size-smalls);
+  font-weight: 400;
+  line-height: var(--eighteen-pixel);
+  letter-spacing: var(--letter-spacing-negative);
+  text-align: left;
+  color: var(--color-gray);
+}
+.purchase-summary__title{
+  font-size: var(--twenty-pixel);
+  font-weight: var(--font-weight-small);
+  line-height: 30px;
+  letter-spacing: var(--letter-spacing-positive);
+  text-align: left;
+  color: var(--color-gray);
+  padding: var(--twenty-pixel);
+}
+
+.purchase-summary__analytics{
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  max-width: 100%;
+  margin: 0px 20px 20px 20px;
+}
+
+.purchase-summary__analytics__link {
+  font-size: 0;
+  inset: 0;
+  position: absolute;
+  z-index: 1;
+}
+
+.purchase-summary__analytics-completed{
+  display: flex;
+  flex-direction: column;
+  padding: var(--zero);
+  position: relative;
+}
+
+.purchase-summary__analytics-completed:hover {
+  box-shadow: var(--box-shadowe);
+}
+
+.purchase-summary__analytics-revision{
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  width: var(--width);
+  background-color: var(--color-white);
+  padding: 25px;
+}
+
+.purchase-summary__analytics-bord{
+  border: 1px solid var(--color-solid);
+  margin-top: 8px;
+}
+
+.purchase-summary__analytics-img{
+  width: var(--width);
+  height: auto;
+  margin-left: 0;
+}
+
+.purchase-summary__analytics-text,
+.purchase-summary__analytics-code,
+.purchase-summary__analytics-title{
+  font-size: 16px;
+  font-weight: var(--font-weight-small);
+  line-height: 24px;
```

Comment:
El diseño dice otra cosa
<img width="579" alt="image" src="https://github.com/user-attachments/assets/1317e9f6-9b07-4ccd-8ca8-eda97a45796a">
```suggestion
  font-size: 12px;
  font-weight: var(--font-weight-small);
  line-height: 16px;
```